homeassistant/components/arcam_fmj/media_player.py

```diff
@@ -96,12 +96,12 @@ def state(self) -> MediaPlayerState:
         return MediaPlayerState.OFF
 
     @property
-    def device_info(self):
+    def device_info(self) -> DeviceInfo:
         """Return a device description for device registry."""
         return DeviceInfo(
             identifiers={
                 (DOMAIN, self._uuid),
-                (DOMAIN, self._state.client.host, self._state.client.port),
+                (DOMAIN, self._state.client.host, self._state.client.port),  # type: ignore[arg-type]
```

Comment:
Would be nice to migrate the identifier in a follow up.

Reply:
Yup. I had forgotten this integration used old format. So was good to find it out.